bindings/python/test.py

```diff
@@ -266,6 +268,34 @@ def test_apply_settings(self):
 		self.assertEqual(s.get_settings()['num_want'], 66)
 		self.assertEqual(s.get_settings()['user_agent'], 'test123')
 
+class test_example_client(unittest.TestCase):
+
+	def test_execute_client(self):
```

Comment:
does this actually work? it just runs the program for a few seconds and make sure it shuts down cleanly?